srcpkgs/eiskaltdcpp-qt/patches/musl.patch

```diff
@@ -1,20 +1,33 @@
---- eiskaltdcpp-qt/src/main.cpp	2015-04-20 02:57:16.000000000 +0500
-+++ eiskaltdcpp-qt/src/main.cpp.new	2018-01-07 08:47:19.501488838 +0500
-@@ -72,7 +72,7 @@
- #if !defined(Q_WS_WIN)
+https://github.com/eiskaltdcpp/eiskaltdcpp/commit/fc9bc0c2
```

Comment:
For the record, when dealing with execinfo.h, it's a bit more correct to check for glibc, since execinfo.h is not a standard. You will have to add any new platform you port this to the list of conditionals you check, instead of just potentially missing some functionality there. Even better would be to see if the function exists, which should be reaasonably simple, given that you are using CMake. I have an example of checking for a function here: https://github.com/MediaArea/ZenLib/pull/126

Reply:
Maybe in the future versions. But not for now.